tsconfig.json

```diff
@@ -26,16 +26,21 @@
     "allowJs": true,
     "checkJs": true,
     "paths": {
-      "$lib": ["src/lib"],
-      "$lib/*": ["src/lib/*"],
-      "$common": ["src/common"],
-      "$common/*": ["src/common/*"],
-      "$cli": ["src/cli"],
-      "$cli/*": ["src/cli/*"],
-      "$server": ["src/server"],
-      "$server/*": ["src/server/*"]
+      "$web-local/lib": ["web-local/lib"],
+      "$web-local/lib/*": ["web-local/lib/*"],
+      "$web-local/common": ["web-local/common"],
+      "$web-local/common/*": ["web-local/common/*"],
+      "$web-local/cli": ["web-local/cli"],
+      "$web-local/cli/*": ["web-local/cli/*"],
+      "$web-local/server": ["web-local/server"],
+      "$web-local/server/*": ["web-local/server/*"]
```

Comment:
Why do we use `$` over `@`? And what do you think about prefix the paths with `@rilldata` for clarity? Like `import { ... } from "@rilldata/web-common"`. (Prisma also does that: https://github.com/prisma/prisma/blob/main/tsconfig.build.bundle.json)

And is it necessary to duplicate each path with a `/*` variant?

Reply:
The standard to use $ instead of @ comes from sveltekit. Adding @rilldata will possibly conflicts with actual packages. I am not sure about the duplication, it was added by sveltekit init I believe (was present before i joined the old data-modeler).

Reply:
Looked a little more into this. `@rilldata` should be a [scope](https://docs.npmjs.com/cli/v8/using-npm/scope) that only we can publish to, so conflicts shouldn't be a problem.

The idea of using NPM workspaces is to treat each workspace as a separate package, but with the convenience of paths resolving locally instead of through NPM (though you can also publish them to NPM). Check out these examples:

- [grafana/packages](https://github.com/grafana/grafana/tree/main/packages) (or for a single package: [grafana/packages/grafana-ui](https://github.com/grafana/grafana/tree/main/packages/grafana-ui))
- [prisma/packages](https://github.com/prisma/prisma/tree/main/packages)